pkg/domain/entities/system.go

```diff
@@ -0,0 +1,13 @@
+package entities
+
+import (
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+type ServiceOptions struct {
```

Comment:
Can we add comments to this type? Once we enable `golint` it would complain about it.

Reply:
done.